base/libgit2/remote.jl

```diff
@@ -8,6 +13,12 @@ function GitRemote(repo::GitRepo, rmt_name::AbstractString, rmt_url::AbstractStr
     return GitRemote(repo, rmt_ptr_ptr[])
 end
 
+"""
+    GitRemote(repo::GitRepo, rmt_name::AbstractString, rmt_url::AbstractString, fetch_spec::AbstractString) -> GitRemote
+
+Look up a remote git repository using its name and url. Also provide specifications
+for how to perform fetches from the remote, for instance which default branch to fetch from.
```

Comment:
IMO the wording is a little confusing. I would say something more along the lines of

> Look up a remote git repository using the repository's name and URL, as well as specifications for how to fetch from the remote (e.g. which remote branch to fetch from).

Similarly it should be "remote branch," not "default branch," right? Isn't there only one default branch in a repository?